src/main/java/com/booksaw/betterTeams/message/MessageManager.java

```diff
@@ -71,8 +116,12 @@ public static void setLanguage(String lang) {
 	 * @param configManager the configuration manager
 	 */
 	public static void addMessages(@NotNull ConfigManager configManager) {
-		prefix = ChatColor.translateAlternateColorCodes('&',
-				Objects.requireNonNull(Main.plugin.getConfig().getString("prefixFormat")));
+		// prefix =
+		// Objects.requireNonNull(Main.plugin.getConfig().getString("prefixFormat"));
```

Comment:
Remove commented code, defaulting the value as you did below is an improvement :)